doc/nrf/ug_nrf9160.rst

```diff
@@ -121,28 +117,23 @@ Delta patches
 Band lock
 *********
 
-The band lock is a functionality of the application that lets you send an
-AT command to the modem instructing it to operate only on specific bands.
-Band lock is handled by the **LTE Link Control** driver and is by default
-disabled in its Kconfig file.
+The band lock is a functionality of the application that lets you send an AT command to the modem instructing it to operate only on specific bands.
+The band lock is handled by the **LTE Link Control** driver.
+By default, it is disabled in its Kconfig file.
 
-The modem can only operate on four certified bands: 3, 4, 13, and 20.
-The application cannot override this restriction.
+The modem can operate in the E-UTRA Bands: 1, 2, 3, 4, 5, 8, 12, 13, 17, 18, 19, 20, 25, 26, 28 and 66.
 
-You can, however, use the band lock to restrict modem operation to a subset of
-the four bands, which might improve the performance of your application.
-To check which bands are certified in your region,
-visit `nRF9160 Certifications`_.
+You can use the band lock to restrict modem operation to a subset of the supported bands, which might improve the performance of your application.
+To check which bands are certified in your region, visit `nRF9160 Certifications`_.
 
-To set the band lock, edit the file :file:`drivers/lte_link_control/lte_lc.c`
-and modify the ``static const char lock_bands`` define.
-Each bit in this define represents one band counting from 1 to 20.
-Therefore, the default setting for bands 3, 4, 13, and 20 is represented by::
+The band lock is a non-volatile setting that must be set before activating the modem.
+It disappears when the modem is reset.
 
-	10000001000000001100
+To set the band lock, edit the file :file:`lib/lte_link_control/lte_lc.c` and modify the ``static const char lock_bands`` define.
+Each bit in this define represents one band.
+Its maximum length is 88 characters.
```

Comment:
```suggestion
Maximum length of each band is 88 characters.
```

Reply:
[Bit string, 88 bits.](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fref_at_commands%2FREF%2Fat_commands%2Fmob_termination_ctrl_status%2Fxbandlock_read.html&resultof=%22%62%61%6e%64%6c%6f%63%6b%22%20)

Reply:
Thanks, I rephrased the sentence a bit.